src/server/util/verify_env.js

```diff
@@ -0,0 +1,17 @@
+module.exports = () => {
+  if (!process.env.GITHUB_KEY) {
+    throw new Error('Missing env variable GITHUB_KEY!');
+  }
+
+  if (!process.env.GITHUB_SECRET) {
+    throw new Error('Missing env variable GITHUB_SECRET!');
+  }
+
+  if (!process.env.SESSION_SECRET) {
+    throw new Error('Missing env variable SESSION_SECRET!');
+  }
+
+  if (!process.env.HOST) {
+    throw new Error('Missing env variable HOST');
+  }
+};
```

Comment:
Feels like it doesn't need to be in a separate file. Also, this is repetitive/copy-paste, try to abstract a bit